### PR TITLE
node promise

### PR DESCRIPTION
**Pull request by jckhang**
## What does this PR do?
1. downward support old node version
For node version 12.X.X, the way to import and use promises is 
```
import { promises as fpm } from 'fs'
fpm.readFile(...)
```

The current syntax only works for 13.2.X+, and 14.X.X 
```
import { readFile } from 'fs/promises'
readFile(...)
```

2. update pretty function

new argument
pretty_method: accern, dot
dot_output: svg, ascii, dot

## Changes

-  [x] Use old syntax
-  [x] New ts package vesion

## Before reviewing

-   [x] Are there any linked PRs?
-   [ ] Did you test this PR?
-   [ ] Is there any migration or seeding necessary?

### Links:

-   xyme-backend: https://github.com/Accern/xyme-backend/pull/119

### CODE REVIEW

File: packages/python/accern_xyme/util.py [hunk 1]
```diff
@@ -168,6 +168,17 @@ def is_jupyter() -> bool:
...
 
 
+def has_graph_easy() -> bool:
+    import subprocess
+
+    try:
+        subprocess.Popen(["graph-easy", "--help"])
+        return True
+    except FileNotFoundError:
+        print({"filenot fiound"})
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/util.py:178]: ? why set? also typo. also which file? better error message please

File: packages/python/accern_xyme/accern_xyme.py [hunk 2]
```diff
@@ -1234,20 +1236,60 @@ def get(hnd: 'ComputationHandle') -> ByteResponse:
...
+        else:
+            from graphviz import Source
+
+            graph = Source(graph_str)
+            if dot_output == "dot":
+                return graph_str
+            if dot_output == "svg":
+                if not is_jupyter():
+                    print(
+                        f"Warning: Ipython instance not found. \n{graph_str}")
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/accern_xyme.py:1270]: space before \n?

File: packages/python/accern_xyme/accern_xyme.py [hunk 3]
```diff
@@ -435,11 +437,11 @@ def check_error(req: Response) -> None:
         if method == METHOD_GET:
             req = requests.get(url, params=args, headers=headers)
             check_error(req)
-            return StringIO(req.text)
+            return StringIO(f"{req.text}")
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/accern_xyme.py:440]: does this hide errors?

File: packages/python/accern_xyme/accern_xyme.py [hunk 4]
```diff
@@ -1234,20 +1236,60 @@ def get(hnd: 'ComputationHandle') -> ByteResponse:
...
+                        f"{graph_str}")
+                else:
+                    import subprocess
+                    cmd = ["echo", graph_str]
+                    p1 = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+                    p2 = subprocess.check_output(
+                        ["graph-easy"], stdin=p1.stdout)
+                    print(p2.decode("utf-8"))
+            else:
+                raise ValueError("invalid dot output option, use svg or ascii")
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/accern_xyme.py:1291]: missing dot

File: packages/python/accern_xyme/accern_xyme.py [hunk 5]
```diff
@@ -1234,20 +1236,70 @@ def get(hnd: 'ComputationHandle') -> ByteResponse:
...
+                        print(svg_str)
+                    else:
+                        from IPython.display import display as idisplay
+                        from IPython.display import SVG
+                        idisplay(SVG(svg_str))
+                else:
+                    return svg_str
+            if dot_output == "ascii":
+                if not has_graph_easy():
+                    return render(graph_str)
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/accern_xyme.py:1289]: warning?

**Review by JosuaKrause — APPROVED** [review]
lgtm

File: packages/python/accern_xyme/accern_xyme.py (hunk 3 above)
Comment (jckhang) [reply packages/python/accern_xyme/accern_xyme.py]: reverted

File: packages/python/accern_xyme/accern_xyme.py (hunk 5 above)
Comment (jckhang) [reply packages/python/accern_xyme/accern_xyme.py:1289]: waring is in the util function

**Pull request merged**